source/common/formatter/substitution_formatter.cc

```diff
@@ -463,6 +465,11 @@ SubstitutionFormatParser::getKnownFormatters() {
          [](const std::string& format, const absl::optional<size_t>& max_length) {
            return FilterStateFormatter::create(format, max_length, true);
          }}},
+      {"PROXY_PROTOCOL_TLVS",
+        {CommandSyntaxChecker::PARAMS_REQUIRED,
```

Comment:
[q] I made the `tlv_type` param required, but we could also make it optional and parse the entire TLV vector if the param is not provided. I can add that functionality if it would be useful.

Reply:
For our use case, this way works, LGTM.